docs/cn/FAQ/Protoc-Plugin-Fails-When-Build-CN.md

```diff
@@ -0,0 +1,8 @@
+现象：
+- Collector启动时加载protoc-plugins插件产生“另一个程序正在使用此文件，进程无法访问”的错误。
```

Comment:
I doubt this is happening when collector starts, I think this is in the process of maven build.

Reply:
Ok, that's changed.